### DISCUSSION TITLE
Hackathon partner

### DISCUSSION THREAD

**Discussion by A-Nasser-Unity** [MiChaelinzo/AR-Treasure-Quest#18] [General]
Hello i,m a game developer and designer using Unity engine since 2014.
Saw your post on Snap Lens hackathon.
I’m searching for a teammate for my lens idea and for other gaming hackathons.

**Comment by MiChaelinzo** [comment]
Awesome let's connect and collaborate!

**Comment by A-Nasser-Unity** [comment]
Thanks for the quick reply
Do you use discord
My discord is: A.Nasser#3247

بتاريخ الخميس، 12 يناير 2023، جاء من Michael G. Inso <
***@***.***>:
